security/openpace/Portfile

```diff
@@ -12,6 +12,7 @@ maintainers             {lbschenkel @lbschenkel} openmaintainer
 description             Cryptographic library for EAC version 2
 long_description        OpenPACE implements Extended Access Control (EAC) \
                         version 2 as specified in BSI TR-03110.
+                        revision 1
```

Comment:
Indented by mistake here.

Reply:
Good point, that one's a corner case that my sed magic didn't catch and probably needs to be fixed manually.